src/server/setcaps-timed.sh

```diff
@@ -4,4 +4,4 @@ if [ "$MIC_RUN" != "" ]; then
 	exit 1
 fi
 
-setcap cap_sys_time+ep /usr/bin/timed-qt5
```

Comment:
Could you still fix the echo line above?

echo "setcaps-timed-qt5.sh - returning FAIL to postpone oneshot to first boot"
=>
echo "setcaps-timed.sh - returning FAIL to postpone oneshot to first boot"

Thank you! We're almost there.